config/routes.rb

```diff
@@ -1,29 +1,97 @@
 require 'sidekiq/web' if defined? Sidekiq
 require Rails.root.join('lib', 'router_constraints.rb')
 
-Ontohub::Application.routes.draw do
+Specroutes.define(Ontohub::Application.routes) do
 
   resources :filetypes, only: :create
 
   # IRI Routing #
   ###############
-  #
+  # as per Loc/Id definition
+
+  # Special (/ref-based) Loc/Id routes
+  specified_get '/ref/:reference/:repository_id/*locid' => 'ontologies#show',
+    as: :ontology_iri_versioned,
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.